rollup-interface/src/state_machine/stf.rs

```diff
@@ -86,22 +82,23 @@ pub trait StateTransitionFunction<Vm: Zkvm> {
     /// Perform one-time initialization for the genesis block.
     fn init_chain(&mut self, params: Self::InitialState);
 
-    /// Called at the beginning of each DA-layer block - whether or not that block contains any
+    /// Called at the beginning of each **DA-layer block** - whether or not that block contains any
     /// data relevant to the rollup.
-    /// If slot is started in Node context, default witness should be provided
-    /// if slot is tarted in Zero Knowledge context, witness from execution should be provided
+    /// If slot is started in Full Node mode, default witness should be provided
+    /// if slot is started in Zero Knowledge mode, witness from execution should be provided
     fn begin_slot(&mut self, witness: Self::Witness);
 
     /// Apply a blob/batch of transactions to the rollup, slashing the sequencer who proposed the blob on failure.
     /// The concrete blob type is defined by the DA layer implementation, which is why we use a generic here instead
     /// of an associated type.
+    /// TODO: Misbehaviour_hint : what it is ?
```

Comment:
@bkolad @preston-evans98 what is purpose of this variable, and it is not used currently.